cmd/get/get.go

```diff
@@ -57,3 +153,101 @@ func toPrintableProjects(projects []*admin.Project) []interface{} {
 
 	return res
 }
+
+func toPrintableTask(tasks []*admin.NamedEntityIdentifier) []interface{} {
+	type printableTasks struct {
+		Name    string `header:"Name"`
+		Project string `header:"Project"`
+		Domain  string `header:"Domain"`
+	}
+
+	res := make([]interface{}, 0, len(tasks))
+	for _, p := range tasks {
+		res = append(res, printableTasks{
+			Name:    p.Name,
+			Domain:  p.Domain,
+			Project: p.Project,
+		})
+	}
+
+	return res
+}
+
+func toPrintableGetTask(tasks []*admin.Task) []interface{} {
+	type printableTask struct {
+		Version    string `header:"Version"`
```

Comment:
linter seems off, I think we should add the same linting rules to this repo as our other repos'

Reply:
ok